tests/integration/assessment/test_workflows.py

```diff
@@ -5,7 +5,7 @@
 from databricks.sdk.service.iam import PermissionLevel
 
 
-@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=8))
+@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=25))
```

Comment:
timeout too long

Reply:
changed to 15. With the linting, 8 minutes almost always fails

Reply:
15 is not enough, using 20

Reply:
same way as we do with schemas, add `include_jobs` / `include_dashboards` into `WorkspaceConfig`, so that it lints only what we want in scope of the integration test. we can't have this test to run longer than 9 minutes.